packages/environment-check/src/checks/get-installed.ts

```diff
@@ -152,3 +152,22 @@ export async function getFioriGenVersion(): Promise<string> {
     }
     return fioriGenVersion;
 }
+
+/**
+ * Returns the versions of node.js modules.
+ *
+ * @returns modules and versions
+ */
+export async function getProcessVersions(): Promise<{ [id: string]: string }> {
```

Comment:
Can we return `NodeJS.ProcessVersions` here?

```suggestion
export async function getProcessVersions(): Promise<NodeJS.ProcessVersions> {
```

Reply:
With the current method we don't know the properties returned, so I've used a type assertion on the returned object